src/init.cpp

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
-// Copyright (c) 2009-2014 The Bitcoin Core developers
```

Comment:
Oh I see now, my commits are credited in there, I merely didn't know where to look (I failed to grep -R myself in the tree), thank you